src/ClassicUO.IO/UOFile.cs

```diff
@@ -104,14 +104,31 @@ private void Load()
             }
         }
 
-        public virtual void Dispose()
+        protected override void Dispose(bool disposing)
         {
+            bool hasAcquiredPointer = StartAddress != IntPtr.Zero;
+
+            base.Dispose(disposing);
+
 #if USE_MMF
```

Comment:
TODO: Should we even have this? Is there a situation where we don't want MMF? I am thinking we can simplify the code assuming this is always true and removing the pragma.

Reply:
I was wondering the same. The "no memory map" case isn't implemented anyway, so we should just remove the `#if`.